pkg/motor/motor.go

```diff
@@ -73,7 +52,7 @@ func EmptyMotor(id string) *motorNodeImpl {
 
 func initMotor(mtr *motorNodeImpl, options ...mpc.WalletOption) (err error) {
 	// Create Client instance
-	mtr.Cosmos = client.NewClient(client.ConnEndpointType_BETA)
+	mtr.Cosmos = client.NewClient(client.ConnEndpointType_LOCAL)
```

Comment:
? Maybe let's not hardcode this?

Reply:
.. this should had been reverted. you are right ill create an issue